src/org/usfirst/frc4904/sovereignty/FusibleNavX.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+import org.usfirst.frc4904.standard.custom.sensors.NavX;
+import edu.wpi.first.wpilibj.SerialPort.Port;
+
+public class FusibleNavX extends NavX implements Fusible<Double> {
```

Comment:
I do not think it would be problematic to add the Fusible interface directly to the NavX; "NavX" is already a subclass we made mostly in order to implement various custom interfaces.